Duckling/Numeral/DE/Rules.hs

```diff
@@ -323,15 +215,25 @@ ruleIntegerWithThousandsSeparator = Rule
       _ -> Nothing
   }
 
+ruleAllNumeralWords :: Rule
+ruleAllNumeralWords = Rule
+  { name = "simple and complex numerals written as one word"
+  , pattern = [regex "(ein|zwei|drei|vier|fünf|sech|sieb|acht|neun|zehn|elf|zwölf|hundert|tausend)?([^\\s]+)?(eine[m|n|r|s]?|eins?|zwei|drei|vier|fünf|sechs|sieben|acht|neun|zehn|elf|zwölf|[s|ß|z]ig|hundert|tausend)"]
+  , prod = \tokens -> case tokens of
+      (Token RegexMatch (GroupMatch matches):_) ->
+        case parseNumeral $ concat $ Text.unpack . Text.toLower <$> matches of
+          Just n  -> integer n
+          Nothing -> Nothing
```

Comment:
```suggestion
maybe
  Nothing integer
  (parseNumeral $ concat $ Text.unpack . Text.toLower <$> matches)
```

Reply:
`maybe Nothing f` is just `fmap f`

Reply:
I adopted the combination of the suggestion by @stroxler and @chessai.